spec/validators/date_not_in_future_validator_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe DateNotInFutureValidator do
+  subject do
+    Class.new {
+      include ActiveModel::Validations
+      attr_accessor :date
+      validates :date, date_not_in_future: true
+
+      def self.name
+        "Transaction"
+      end
+    }.new
+  end
+
+  describe "when date is nil" do
```

Comment:
As these blocks are adding new context I think we could rename them to `context`. I suppose the method we are describing is `valid?`?